ociclient/test/envtest/envtest.go

```diff
@@ -235,6 +243,8 @@ func (e *Environment) runRegistry(ctx context.Context) error {
 	e.Addr = e.RegistryConfiguration.HTTPConfig.Addr
 	go func() {
 		defer func() {
+			e.mu.Lock()
```

Comment:
Wouldn't use defer here to unlock the mutex, as in the assignment to e.stopped no possible error case is thrown. Therefore would go for better readability

```suggestion
			e.mu.Lock()
			e.stopped = true
			e.mu.Unlock()
```